sphinx/source/index.rst.txt

```diff
@@ -0,0 +1,45 @@
+.. _about:
+
+Bokeh is an interactive visualization library for modern web browsers. It
+provides elegant, concise construction of versatile graphics, and affords
+high-performance interactivity over large or streaming datasets. Bokeh can help
+anyone who would like to quickly and easily make interactive plots, dashboards,
+and data applications.
+
+To get started using Bokeh to make your visualizations, begin with the
+:ref:`userguide`.
+
+For examples of how you might use Bokeh with your own data, peruse
+the :ref:`gallery`.
+
+For detailed information about specific Bokeh components, consult the
+:ref:`refguide`.
+
+If you are interested in contributing to Bokeh, or extending the library,
+check out the :ref:`devguide` Guide.
+
+If you'd like to search for a particular topic, use the search box below:
+
+.. raw:: html
+
+    <form class="bd-search align-items-center" action="search.html" method="get">
+      <input type="search" class="form-control" name="q" id="search-input" placeholder="&#x1F50D; Search the docs ..." aria-label="Search the docs ..." autocomplete="off">
+    </form>
```

Comment:
BTW if you know how to make this search field not be so wide, I couldn't figure it out last time I tried :)

Reply:
@bryevdv can do it. Will make the necessary changes!

Reply:
@bryevdv I changed the width, and centred the search bar. Looks much better now!